README.md

```diff
@@ -22,7 +22,7 @@ To have your sources automatically formatted on each build, add to your pom.xml:
     <build>
         <plugins>
             <plugin>
-                <groupId>com.coveo</groupId>
+                <groupId>com.spotify</groupId>
```

Comment:
nitty-pick: What do you think about adding `fmt` or `plugin` to the end of `groupId`? So that we can group it under an umbrela instead of having it together with things that are not necessarily realted.

Reply:
Need to find out if we are allowed to deploy to com.spotify.fmt

Reply:
The only one in maven central is `com.spotify.docgenerator`

Reply:
I think we should change the package name as well to not add too much to the `com.spotify` space.

Reply:
yep, I believe we change everything to reflect it